### PR TITLE
Feature: specs

### PR DESCRIPTION
**Pull request by inkel**
Yes, I'm using the stdlib `Minitest`, because why not?

These aren't probably right, I'm mixing the spec syntax with the unit test syntax, and I don't care. I'm free, I'm happy, I'm the owner.

### CODE REVIEW

**Comment by inkel** [comment]
image

File: spec/disyuntor_spec.rb [hunk 1]
```diff
@@ -0,0 +1,308 @@
+require "minitest/autorun"
+require_relative "../lib/disyuntor"
+
+describe Disyuntor do
+  let(:circuit) { Disyuntor.new }
+
+  describe :new do
+    it "be closed by default" do
```
Comment (fsaravia) [inline spec/disyuntor_spec.rb]: Missing a "should" here? Otherwise maybe it should read `it "is closed by default"`

**Comment by fsaravia** [comment]
:metal:

File: spec/disyuntor_spec.rb (hunk 1 above)
Comment (inkel) [inline spec/disyuntor_spec.rb]: Definitely.

**Comment by inkel** [comment]
Any other comments about this? I'm not 100% convinced about the format, however, any other test that I write leaves me with a similar sensation, so I think I'd rather have a pre-release with tests already and then work on improving them. Thoughts?

**Comment by fsaravia** [comment]
I like it the way it is, I have the feeling that the tests can be organized a little better but have no ideas for that at the moment, they look good enough to me for merging and they can be optimized later.

**Comment by inkel** [comment]
@fsaravia I'm going to take your word as a :+1: and held you responsible for saying it :stuck_out_tongue_winking_eye:

**Pull request merged**